src/modules/admin/organisations/services/organisations.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { get } from 'lodash';
+
+import { GetOrganisationDocument } from '~generated/graphql-db-types-hetarchief';
+import { ORGANISATION_QUERIES } from '~modules/admin/organisations/organisations.consts';
+import {
+	GqlOrganisation,
+	Organisation,
+	OrganisationQueries,
+} from '~modules/admin/organisations/organisations.types';
+import { DataService } from '~modules/data/services/data.service';
+
+@Injectable()
+export class OrganisationsService {
+	private logger: Logger = new Logger(OrganisationsService.name, { timestamp: true });
+	private queries: OrganisationQueries;
+
+	constructor(private configService: ConfigService, protected dataService: DataService) {
+		this.queries = ORGANISATION_QUERIES[configService.get('avoOrHetArchief')];
```

Comment:
```suggestion
		this.queries = ORGANISATION_QUERIES[this.configService.get('avoOrHetArchief')];
```

Reply:
beide zijn correct, aangezien we daar in de constructor zitten en dus rechtstreeks toegang hebben tot die values.
Maar voor consistency is het wel beter met .this